docker/.dockerignore

```diff
@@ -16,3 +16,5 @@
 
 # node_modules in app root weights over 130 MB
 app/node_modules
+
+app/docker
```

Comment:
I can see nginx config in docker folder
aren't they needed?

Reply:
I will check